docs/guide/i18n.md

```diff
@@ -22,12 +22,14 @@ export default defineConfig({
   locales: {
     root: {
       label: 'English',
-      lang: 'en'
+      lang: 'en',
+      languageButtonAriaLabel: 'Change Language'
```

Comment:
It would be cool if we can have shorter name for this. Maybe `langMenuAriaLabel`?

Reply:
Thanks for the good suggestion. I've updated the commits accordingly.